packages/api/src/index.ts

```diff
@@ -85,18 +85,24 @@ export type {
 } from './interfaces/endpoints/entitlement.input';
 
 export { HelixEventSubApi } from './endpoints/eventSub/HelixEventSubApi';
+export { HelixEventSubConduit } from './endpoints/eventSub/HelixEventSubConduit';
+export { HelixEventSubConduitShard } from './endpoints/eventSub/HelixEventSubConduitShard';
 export { HelixEventSubSubscription } from './endpoints/eventSub/HelixEventSubSubscription';
 export { HelixPaginatedEventSubSubscriptionsRequest } from './endpoints/eventSub/HelixPaginatedEventSubSubscriptionsRequest';
 export type {
 	HelixEventSubTransportData,
+	HelixEventSubConduitShardData,
```

Comment:
Please don't export external data interfaces here unless they are part of a publicly exposed parameter type, return type (both of these cases should be avoided though) or used in another Twurple package.